cfme/utils/ssh.py

```diff
@@ -646,6 +646,27 @@ def patch_file(self, local_path, remote_path, md5=None):
             raise Exception("Unable to patch file {}: {}".format(remote_path, result.output))
         return True
 
+    def is_file_available(self, remote_path):
+        """Check file available or not on remote host
+
+        Args:
+            remote_path: Path to remote file
+
+        Returns:
+            True if file available else False
+        """
+        files = self.run_command(f"ls {os_path.dirname(remote_path)}").output
+        return os_path.basename(remote_path) in files
+
+    def remove_file(self, remote_path):
+        """Remove file from remote host
+
+        Args:
+            remote_path: Path to remote file
+        """
+        if self.is_file_available(remote_path):
+            self.run_command(f"rm -rf {remote_path}")
```

Comment:
I don't think `-r` should be included here, as its supposed to remove_file, not recursively delete an entire path (if passed).  At minimum it should be a kwarg.

Reply:
Pushed a commit for this, adding force and recursive as kwargs.